<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdaptBrowserViewWidget</class>
 <widget class="QWidget" name="AdaptBrowserViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="labels_checkbox_">
     <property name="text">
      <string>Labels</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="y_axis_checkbox_">
     <property name="text">
      <string>YAxis</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="x_axis_checkbox_">
     <property name="text">
      <string>XAxis</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="zero_fitted_">
       <property name="text">
        <string>0-Line Fitted</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zero_centered_">
       <property name="text">
        <string>0-Line Centered</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
